Resources/Server Info/Guidebook/Med/Chemistry.xml

```diff
@@ -0,0 +1,38 @@
+﻿<Document>
+# Chemistry
+
+Chemistry is a subsection of the Medical department, but also the science of combining reagents together to make a product. What you know of real-life chemistry and pharmacology may be throw to the wind; this is nothing like it.
+
+## Basic tools
+Your most basic tool for chemistry will be your trusty beaker. These most commonly come in 2 sizes, normal and large. They hold 50u and 100u of reagents, respectivley.
+<Box>
+<GuideEntityEmbed Entity="Beaker"/>
+<GuideEntityEmbed Entity="LargeBeaker"/>
+</Box>
+These hold all your reagents, be it a raw material like [color=#434b4d]iron[/color], a medicine like [color=#ffaa00]Bicardine[/color], or a weapon like [color=#fa00af]napalm[/color]. Careful not to drop them, as they are made out of glass and can shatter easily when thrown or dropped. Anything inside the beaker will be spilled on the floor if the beaker breaks, so if you're working with a potentially dangerous chemical like [color=#a1000b]polytrinic acid[/color], keep an eye on glimmer!
```

Comment:
```suggestion
These hold all your reagents, be it a raw material like [color=#434b4d]iron[/color], a medicine like [color=#ffaa00]bicaridine[/color], or a weapon like [color=#fa00af]napalm[/color]. Careful not to drop them, as they are made out of glass and can shatter easily when thrown or dropped. Anything inside the beaker will be spilled on the floor if the beaker breaks, so if you're working with a potentially dangerous chemical like [color=#a1000b]polytrinic acid[/color], keep an eye on glimmer!
```

Reply:
"keep an eye on glimmer" requires context here. I know what it means, but someone who is reading this as a first time player won't.